bsp/boards/i2c.h

```diff
@@ -33,10 +33,10 @@ void i2c_write_register(uint8_t bus_num, uint8_t slave_addr, uint8_t length, uin
 uint8_t i2c_slave_present(uint8_t bus_num,uint8_t slave_address);
 
 void i2c_read_byte(uint8_t address, uint8_t* byte);
-void i2c_read_bytes(uint8_t address, uint8_t* buffer, uint32_t length);

```

Comment:
@peretuset, I know that the I2C drivers is only implemented of the OpenMoteCC2538, but I'd like to understand what this new signature does. That is, what is the returned value?
You might simply document the definition.